app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -47,7 +47,7 @@ def service_provider_homepage_url
       end
 
       def outage_message_enabled?
-        IdentityConfig.store.in_person_outage_message_enabled == true && outage_dates_present?
+        IdentityConfig.store.in_person_outage_message_enabled && outage_dates_present?
```

Comment:
This isn't strictly in scope for the ticket, but I thought it was a good piece of clean-up to do.